Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+// define a tag according to the Docker tag rules https://docs.docker.com/engine/reference/commandline/tag/
```

Comment:
We will need to read/create this file in each branch we want Jenkins to build.  We will likely make this once in master (so branches off of master will have it to) and never touch it again.